repo/packages/B/beta-tensorflow/1/config.json

```diff
@@ -0,0 +1,329 @@
+          "type": "integer",
+          "default": 2
+        },
+        "placement_constraint": {
+          "title": "Placement constraint",
+          "description": "Marathon-style placement constraint for nodes. Example: 'hostname:UNIQUE'",
+          "type": "string",
+          "default": ""
+        },
+        "port": {
+          "title": "Port",
+          "description": "The port Parameter Servers will use for gRPC.",
+          "type": "integer",
+          "default": 2223
+        },
+        "cpus": {
+          "title": "CPU count",
+          "description": "TF PS CPU requirements",
+          "type": "number",
+          "default": 1.0
+        },
+        "mem": {
+          "title": "Memory size (MB)",
+          "description": "TF PS mem requirements (in MB)",
+          "type": "integer",
+          "default": 4096
+        },
+        "disk": {
+          "title": "Disk size (MB)",
+          "description": "TF PS persistent disk requirements (in MB)",
+          "type": "integer",
+          "default": 4096
+        },
+        "disk_type": {
+          "title": "Disk type [ROOT, MOUNT]",
+          "description": "Mount volumes require preconfiguration in DC/OS",
+          "enum": [
+            "ROOT",
+            "MOUNT"
+          ],
+          "default": "ROOT"
+        }
+      },
+      "required": [
+        "count",
+        "cpus",
+        "mem",
+        "port",
+        "disk",
+        "disk_type"
+      ]
+    },
+    "worker": {
+      "description": "TF Worker configuration properties",
+      "type": "object",
+      "properties": {
+        "count": {
+          "title": "Node count",
+          "description": "Number of TF Workers to run",
+          "type": "integer",
+          "default": 3
+        },
+        "placement_constraint": {
+          "title": "Placement constraint",
+          "description": "Marathon-style placement constraint for nodes. Example: 'hostname:UNIQUE'",
+          "type": "string",
+          "default": ""
+        },
+        "port": {
+          "title": "Port",
+          "description": "The port Workers (and GPU Workers) will use for gRPC.",
+          "type": "integer",
+          "default": 2222
+        },
+        "cpus": {
+          "title": "CPU count",
+          "description": "TF Worker CPU requirements",
+          "type": "number",
+          "default": 1.0
+        },
+        "mem": {
+          "title": "Memory size (MB)",
+          "description": "TF Worker mem requirements (in MB)",
+          "type": "integer",
+          "default": 4096
+        },
+        "disk": {
+          "title": "Disk size (MB)",
+          "description": "TF Worker persistent disk requirements (in MB)",
+          "type": "integer",
+          "default": 4096
+        },
+        "disk_type": {
+          "title": "Disk type [ROOT, MOUNT]",
+          "description": "Mount volumes require preconfiguration in DC/OS",
```

Comment:
Should we add a link? Also maybe "Type of persistent disk. Note, that Mount Volumes requireconfiguration in DC/OS (https://dcos.io/docs/latest/storage/mount-disk-resources).